hack/docs/instancetypes_gen_docs.go

```diff
@@ -56,7 +60,7 @@ func main() {
 	opts = opts.MustParse()
 	ctx := injection.WithOptions(context.Background(), *opts)
 
-	cp := awscloudprovider.NewCloudProvider(ctx, cloudprovider.Options{})
+	cp := NewAWSCloudProviderForCodeGen()
```

Comment:
Why not inject `ctx` into this function?

Reply:
I originally had it in a different package.  There is zero benefit in this scenario to passing the context in, it's just holding up a PR.